Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM allenai-docker-private-docker.bintray.io/cuda:8
+FROM 896129387501.dkr.ecr.us-west-2.amazonaws.com/infrastructure/aristo/cuda:8
```

Comment:
Have you checked with aristo-eng or techops that this name is good to put in the repo?